test.py

```diff
@@ -1043,9 +1047,7 @@ async def run(self, options: argparse.Namespace) -> Test:
 
         test_path = os.path.join(self.suite.options.tmpdir, self.mode)
         async with get_cluster_manager(self.mode + '/' + self.uname, self.suite.clusters, test_path) as manager:
-            self.args.insert(0, "--run_id={}".format(self.id))
             self.args.insert(0, "--tmpdir={}".format(options.tmpdir))
-            self.args.insert(0, "--mode={}".format(self.mode))
```

Comment:
can't understand the reason for deleting the parameters

Reply:
Ditto